packages/asc-core/src/components/Icon/Icon.ts

```diff
@@ -15,8 +15,6 @@ const Icon = styled.span<Props>`
   height: 20px;
 
   & > svg {
-    width: 20px;
```

Comment:
Is this change related to adding SVGO?

Reply:
Yes and no. It's not directly linked to the svgo, however, I noticed while playing with svgo config, these lines could be removed as they don't do anything (because now we get rid of the width & height attributes)

Reply:
Nice